HW_1/task_2.py

```diff
@@ -0,0 +1,9 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали
+# S журавликов. Сколько журавликов сделал каждый ребенок, если известно,
+# что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+S = int(input('Введите общее количество журавликов: '))
+if not S % 6:
+    n = S // 6
+print('Петя и Сережа сделали по', n, ', а Катя сделала', n*4);
```

Comment:
Тут произойдет ошибка, если условие будет ложным.
```suggestion
print('Петя и Сережа сделали по', n, ', а Катя сделала', n * 4);
```